src/store/reducers/SovereignVenue.ts

```diff
@@ -23,6 +23,8 @@ export const sovereignVenueReducer = (
 ): SovereignVenueState => {
   switch (action.type) {
     case SovereignVenueActionTypes.SET_SOVEREIGN_VENUE:
+      console.log("reducer", action.payload.sovereignVenue);
+
```

Comment:
```suggestion
```

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0e1ea16c81870c898660c5e5b60aa9a143b57c80